### ISSUE TITLE
Publication: Next Stable Version

### ISSUE THREAD

**Issue by PHIROC** [greenshot/greenshot#518]
Dear Greenshot-Team,
 -
I think that I speak for the majority of my colleagues when I say "thank you" for such a small, comprehensive, powerfull and above all, just simple solution like Greenshot. 
We are getting in touch with you because your product enjoys great popularity among our more than 350 colleagues. Therefore, we are very interested in the next stable version of the software. 
Therefore we would like to request information on further development of the programm and a short prospect for the future. 
Since the last blog entry is from 2020 and the last stable version from August 2017, we would be very happy to receive a status update, when to expect an updated to the next stable version.
-
Yours sincerely,
phiroc

**Comment by wrldtvlr** [comment]
I suspect the evolution of Windows Snipping Tool to cover most of Greenshot functionality has put a damper on any further development of Greenshot.

**Comment by Nagidrop** [comment]
I'd like an update for Greenshot status as well. I loved using it, but then sadly got to drop because of its inactive development. In my opinion, the Snipping Tool and alternatives such as ShareX likely made Greenshot obsolete, and due to insufficent funds/time/people, unfortunately it got abandoned since.

**Comment by vivadavid** [comment]
Hi,

I've tried many screenshot programmes and I believe that Greenshot is still, by far, the best. I especially love the annotation features: I often need to mark text, and I easily switch from one colour to another.

I also hope this programme can have a new stable version soon. I keep my fingers crossed!

Thanks in advance!

**Comment by wrldtvlr** [comment]
Looks like there was an attempt to create a more up to date Greenshot, but it's dormant as well https://github.com/greenshot/Greenshot2.Editor

**Comment by jklingen** [comment]
Hi folks,
sorry, everything's going slowly for quite some time now 😕  We just posted a new blog post about the [current status of Greenshot](https://getgreenshot.org/2024/02/11/current-status-greenshot/).
I hope it adds some clarity.

> tl;dr: We are struggling with a lack of time, but are gradually trying to pick up speed again.

**Comment by vivadavid** [comment]
Hi, @jklingen ,

Thanks for informing us about the situation. We understand you take the time you need: we'll be pleased to try the new stable version whenever it's ready. And personally I might have a look at the latest unstable version.

**Comment by jklingen** [comment]
Oh another one of these tickets, just stumbled across it again 🙈 
In case you haven't noticed: Greenshot 1.3 has been released some time ago. You can get it on our[Greenshot download page](https://getgreenshot.org/downloads/).
